src/securejoin.rs

```diff
@@ -543,14 +566,12 @@ pub(crate) fn handle_securejoin_handshake(
                 send_handshake_msg(context, contact_chat_id, "vc-contact-confirm", "", None, "");
                 inviter_progress!(context, contact_id, 1000);
             }
+            Ok(HandshakeMessage::Done)
         }
         "vg-member-added" | "vc-contact-confirm" => {
-            if join_vg {
```

Comment:
Where did this logic go?

Reply:
The returns in this block are `HandshakeMessage::Propagate()` instead of `::Ignore` or `::Done`.  That's how this logic is handled.

Reply:
But where is the condition that distinguish between join_vg true and false?